lib/rubocop/cop/rails/unused_ignored_columns.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop suggests you remove a column that does not exist in the schema from `ignored_columns`.
+      # `ignored_columns` is necessary to drop a column from RDBMS, but you don't need it after the migration
+      # to drop the column. You avoid forgetting to remove `ignored_columns` by this cop.
+      #
+      # @example
+      #   # bad
+      #   class User < ApplicationRecord
+      #     self.ignored_columns = [:already_removed_column]
+      #   end
+      #
+      #   # good
+      #   class User < ApplicationRecord
+      #     self.ignored_columns = [:still_existing_column]
+      #   end
+      #
+      class UnusedIgnoredColumns < Base
+        include ActiveRecordHelper
+
+        MSG = 'Remove `%<column_name>s` from `ignored_columns` because the column does not exist.'
+
+        def_node_matcher :ignored_columns?, <<~PATTERN
```

Comment:
`?` looks unnecessary because a capture value is returned.

```suggestion
        def_node_matcher :ignored_columns, <<~PATTERN
```